src/mlte/negotiation/negotiation_card.py

```diff
@@ -58,18 +58,23 @@ def from_model(model: ArtifactModel) -> NegotiationCard:  # type: ignore[overrid
             model=model.body.model,
         )
 
-    def save_with(self, context: Context, store: Store) -> None:
+    def save_with(
+        self, context: Context, store: Store, *, parents: bool = False
+    ) -> None:
         """
         Save an artifact with the given context and store configuration.
         :param context: The context in which to save the artifact
         :param store: The store in which to save the artifact
+        :param parents: Indicates whether organizational elements for the
+        artifact are created implicitly on write (default: False)
         """
         with ManagedSession(store.session()) as handle:
             handle.write_artifact(
                 context.namespace,
                 context.model,
                 context.version,
                 self.to_model(),
+                parents=parents,
```

Comment:
I guess this will be moved to Artifact with the Value Artifact PR, but I understand why this is here for now.

Reply:
Correct!